src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -64,6 +64,26 @@ internal static SafeSslHandle AllocateSslContext(SslProtocols protocols, SafeX50
                     throw CreateSslException(SR.net_allocate_ssl_context_failed);
                 }
 
+                // TLS 1.3 uses different ciphersuite restrictions than previous versions.
+                // It has no equivalent to a NoEncryption option.
+                if (policy == EncryptionPolicy.NoEncryption)
+                {
+                    if (protocols == SslProtocols.None)
+                    {
+                        protocols = SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12;
+                    }
+                    else
+                    {
+                        protocols &= ~SslState.Tls13Protocol;
```

Comment:
Is it correct that this is using SslState.Tls13Protocol rather than SslProtocols.Tls13?

Reply:
The enum member isn't being added in this change; `SslState` was the first place in this assembly that needed the value, and since this also needed it I just made the const there internal instead of redefining it here.

They're both `12288` (a number I have typed far too many times this week :smile:)